### DISCUSSION TITLE
Commit Convention

### DISCUSSION THREAD

**Discussion by bramburn** [hassanhabib/The-Standard#217] [General]
hi Hassan, would it be worth adding a commit convention section in the standard? it is not defined properly as to what constitute a fix or a code rub.
My understanding from reading some of the commit histories is that a code rub would just be styling and linting the code and spacing.
Would it cover typos or conventions of plurals and singular or would that fall under a fix?

What would be covered as a fix?

**Comment by cjdutoit** [comment]
Documentation can be found here:  https://github.com/hassanhabib/The-Standard-Team/blob/main/4%20Practices/4%20Practices.md
